packages/material-ui-lab/src/TreeItem/TreeItem.js

```diff
@@ -6,41 +6,53 @@ import Typography from '@material-ui/core/Typography';
 import Collapse from '@material-ui/core/Collapse';
 import { withStyles } from '@material-ui/core/styles';
 import { useForkRef } from '@material-ui/core/utils';
+import { useIsFocusVisible } from '../../../material-ui/src/utils/focusVisible';
 import TreeViewContext from '../TreeView/TreeViewContext';
 
-export const styles = {
-  /* Styles applied to the root element. */
-  root: {
-    listStyle: 'none',
-    margin: 0,
-    padding: 0,
-    outline: 0,
-    '&:focus > $content': {
-      outline: 'auto 1px',
+export const styles = function styles(theme) {
+  return {
+    /* Styles applied to the root element. */
+    root: {
+      listStyle: 'none',
+      margin: 0,
+      padding: 0,
+      outline: 0,
+      cursor: 'pointer',
     },
-  },
-  /* Styles applied to the `role="group"` element. */
-  group: {
-    margin: 0,
-    padding: 0,
-    marginLeft: 26,
-  },
-  /* Styles applied to the tree node content. */
-  content: {
-    width: '100%',
-    display: 'flex',
-    alignItems: 'center',
-  },
-  /* Styles applied to the tree node icon and collapse/expand icon. */
-  iconContainer: {
-    marginRight: 2,
-    width: 24,
-    minWidth: 24,
-    display: 'flex',
-    justifyContent: 'center',
-  },
-  /* Styles applied to the label element. */
-  label: {},
+    /* Styles applied to the `role="group"` element. */
+    group: {
+      margin: 0,
+      padding: 0,
+      marginLeft: 26,
+    },
+    /* Styles applied to the tree node content. */
+    content: {
+      width: '100%',
+      display: 'flex',
+      alignItems: 'center',
+      '&:focus': {
+        outline: 'none',
+      },
+      '&$focusVisible': {
+        backgroundColor: theme.palette.grey[200],
+      },
+      '&:hover': {
+        backgroundColor: theme.palette.grey[200],
```

Comment:
?

```suggestion
        backgroundColor: theme.palette.action.hover,
```

Reply:
I remember when it started with a hover interaction 😆 Glad to see it back